.github/pull_request_template.md

```diff
@@ -9,3 +9,4 @@ Changelog must always be updated.
 ### Checklist
 
 - [ ] Update changelog in CHANGELOG.md.
+- [ ] Request review from oncall area, as well as team (e.g: `oncall-kaas-cloud` GitHub group).
```

Comment:
No strong opinion but is it really necessary to have a checkbox if you have added the right reviewers?

Reply:
Possibly not _necessary_, but it does force the point. We can entirely drop this in future if we get into the habit of requesting review from the area for changing alerts.